docs/quick_start.md

```diff
@@ -0,0 +1,84 @@
+# Quick Start
+
+!!! Note
+    Currently, using the OSTIS Metasystem natively on Windows isn't supported.
+
+1. Clone repository of the OSTIS Metasystem:
+   
+    ```sh
+    git clone https://github.com/NikitaZotov/ostis-metasystem
+    cd ostis-metasystem
+    git checkout feat/update_build_system
+    ```
+
+2. Install C++ problem solver:
+
+    ```sh
+    ./scripts/install_cxx_problem_solver.sh
```

Comment:
need to check this after everything is released

Reply:
Yes, not all repo are release yet